docker/build-tools/Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV ISTIO_TOOLS_BRANCH=${ISTIO_TOOLS_BRANCH}
 ENV COUNTERFEITER_VERSION=v6.2.2
 ENV FORTIO_VERSION=v1.3.1
 ENV GCR_AUTH_VERSION=1.5.0
-ENV GO_BINDATA_VERSION=6025e8de665b31fa74ab1a66f2cddd8c0abf887e
+ENV GO_BINDATA_VERSION=v3.1.0
```

Comment:
Does it make sense to use the latest one? (3.1.2)

Reply:
Actually yes. I was being conservative here as `6025e8d` pointed to `v.3.1.0`. I just tested `v3.1.2` for the go-bindata usages in istio/istio. I don't see difference in generated files from `v3.1.0`. So should be fine.